eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -143,6 +144,19 @@ abstract class SafeMojo extends AbstractMojo {
     )
     protected File transpiled;
 
+    /**
+     * Mojo execution timeout in seconds.
+     * Default value is one hour.
+     * @checkstyle VisibilityModifierCheck (10 lines)
+     * @since 0.28.12
+     */
+    @Parameter(
+        property = "eo.timeout",
+        required = true,
+        defaultValue = "3600"
+    )
+    protected int timeout = 3600;
```

Comment:
@volodya-lombrozo Maybe we need to use this parameter only when needed (required = false), instead of using the default value? WDYT?

Reply:
@Graur I don't like situations where we have to check for `null` like `if(timeout==null)`, moreover the default value was one hour, which is enough. But I've tried to apply your suggestion. Please, check.

Reply:
@volodya-lombrozo Yeah, you are right. Looks not good. You can revert this change if you want. It's up to you

Reply:
@volodya-lombrozo Yep, thanks